docs/guides/concepts/logging.md

```diff
@@ -14,12 +14,14 @@ section.
 ### Usage
 
 To receive log events, simply hook the discord client's log method
-to a Task with a single parameter of type [LogMessage]
+to a `Task` with a single parameter of type [LogMessage].
```

Comment:
of type `[LogMessage]`.

is [LogMessage] a link to the docs? Maybe for consistency, I would change it to `LogMessage`

Reply:
Most bracket-contained words are xrefs.